terraform/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "project_id" {
+  type        = string
```

Comment:
I haven't seen this spacing before, where the `=` line up. Is this standard for this type of file? Otherwise we should remove it.

Reply:
Yeah this is the standard format recommended by terraform. We can use **terraform fmt** to format all the files.